.github/workflows/lint.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: -- -D warnings -A deprecated
+          args: -- -D warnings -A deprecated -W clippy::nursery
```

Comment:
add this in clippy config in Cargo.toml or a config file so it applies locally as well (we don't have to pass it in everytime)

Reply:
done